tests/integ/conftest.py

```diff
@@ -192,7 +191,14 @@ def engine(dynamodb, dynamodbstreams, request):
         table_name_template="{table_name}" + request.config.getoption("--nonce")
     )
     yield engine
-    engine.delete(*engine.scan(User))
+    concrete = set(filter(lambda m: not m.Meta.abstract, walk_subclasses(BaseModel)))
```

Comment:
Can you add a comment explaining what this does?  Looks like cleanup between tests so inheritance doesn't fail when you scan/query but I'm not positive.